.markdownlint.jsonc

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
What's this file for?

Reply:
This specifies which [markdown rules](https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md#md013---line-length) should be included/excluded so that an IDE markdown linter will only check those when a new md file is created.
Specifically, I've added HTML code to the README for better style, which violates the rule.